.github/workflows/collab.yml

```diff
@@ -0,0 +1,52 @@
+name: Build Project on Google Collab (Execution)
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest-m
+    container:
+      image: docker://us-docker.pkg.dev/colab-images/public/runtime:latest
```

Comment:
@kp992 this is not fetching the same environment as what is online and I am getting it from here: https://console.cloud.google.com/artifacts/docker/colab-images/us/public/runtime

Reply:
This is tricky, I read it online that `:latest` should be enough to fetch the recent runtime.

Reply:
me too. It is frustrating it isn't exactly the same as google colab. They must deploy from a different runtime image?